programmers/로또.js

```diff
@@ -0,0 +1,21 @@
+function solution(lottos, winNums) {
+    
+    let [zCnt, wCnt] = [0, 0];
+    let [best, worst] = [0, 0];
+    
+    for (let i = 0; i < 6; i++) {
+        if(lottos[i] === 0) zCnt += 1;
+        
+        for (let j = 0; j < 6; j++) {
+            if (lottos[i] === winNums[j]) wCnt += 1;
+        }
+    }
```

Comment:
```suggestion
       if (winNums.includes(lottos[i]) wCnt++;
```
이렇게 그 값이 있는지 boolean값으로 나오는  includes 메서드를 이용해서 간결하게 할 수도 있답니다.